src/transformers/models/albert/modeling_tf_albert.py

```diff
@@ -1058,6 +1080,16 @@ def call(
             attentions=outputs.attentions,
         )
 
+    def serving_output(self, output):
+        hs = tf.convert_to_tensor(output.hidden_states) if self.config.output_hidden_states else None
+        attns = tf.convert_to_tensor(output.attentions) if self.config.output_attentions else None
+
+        return TFMaskedLMOutput(
+            logits=output.logits,
+            hidden_states=hs,
+            attentions=attns,
+        )
```

Comment:
This fits in less than 119 characters with
```
        return TFMaskedLMOutput(logits=output.logits, hidden_states=hs, attentions=attns)
```

Reply:
AH those ones! Sorry I thought you were talking about only the `if self.config.use_cache...` things. Ok nevermind then, I will also take care of this!